### PR TITLE
Terms of use and badges

### PR DESCRIPTION
**Pull request by symroe**

### CODE REVIEW

File: aggregator/templates/includes/badge-card.html [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{% load staticfiles %}
+<div class="card {{ class }}">
+  <img src="{% static path %}">
+  <code>https://developers.democracyclub.org.uk{% static path %}</code>
```
Comment (chris48s) [inline aggregator/templates/includes/badge-card.html]: Any reason not to use `request.META.HTTP_HOST` here? I'm pretty sure we've already done the requisite CloudFront header fiddling to get internal links to work properly.

File: aggregator/templates/terms.html [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+<li>harms Democracy Club’s ability to deliver the data in future (for example, by weakening the trust placed in us by volunteers or electoral administrators);</li>
+<li>harms the perception of Democracy Club as a non-partisan actor;</li>
+<li>harms the perception of the accuracy of Democracy Club’s data;</li>
+<li>breaks the law (e.g. by the use of personal data for purposes other than an activity that supports or promotes democratic engagement, see s.8(e) of the Data Protection Act 2018).  </li>
+</ul>
+
+<p>We aim to be clear and consistent in our application of these terms. In the event of problems brought to our attention, we’ll aim to follow the following steps:</p>
+<ol>
+<li>Democracy Club will raise the issue with the user, remind them of the terms and suggest steps to ameliorate the issue.</li>
+<li>If steps are not taken, or there is no response from a user, the issue will be reviewed by three board members and access may be removed. The executive director(s) has (have) the option to immediately suspend use before the review if they believe significant harm is occuring.</li>
```
Comment (chris48s) [inline aggregator/templates/terms.html]:
```suggestion
<li>If steps are not taken, or there is no response from a user, the issue will be reviewed by three board members and access may be removed. The executive director(s) has (have) the option to immediately suspend use before the review if they believe significant harm is occurring.</li>
```

**Comment by symroe** [comment]
> and rebuild the docs using ./manage.py build_docs

Nope! I still can't install blueprint for $reasons. Are you able to do this?

> The URLs for the images seem like they're the wrong way round. The ones in /white have a black background and the ones in /black have a white background.

This is how Russ sent them over, and I assume he's talking about the text colour? Happy to swap them if you think it's needed, but FWIW this is how it makes sense to me too…

File: aggregator/templates/terms.html [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+<p>If you would like to discuss different terms of service, please get in touch.</p>
+
+<p>Democracy Club reserves the right to withdraw access if the data is used in such a way that:</p>
+<ul>
+<li>misleads voters (for example, an app to show users their nearest polling station, instead of the correct one allocated to them based on their registered address);</li>
+<li>harms Democracy Club’s ability to deliver the data in future (for example, by weakening the trust placed in us by volunteers or electoral administrators);</li>
+<li>harms the perception of Democracy Club as a non-partisan actor;</li>
+<li>harms the perception of the accuracy of Democracy Club’s data;</li>
+<li>breaks the law (e.g. by the use of personal data for purposes other than an activity that supports or promotes democratic engagement, see s.8(e) of the Data Protection Act 2018).  </li>
```
Comment (JoeMitchell) [inline aggregator/templates/terms.html]: Given that twitter chat with Owen B, I wonder whether we remove the e.g. ?

**Comment by JoeMitchell** [comment]
Left one minor comment. Can't see any of the badges, but assume they're okay!

File: aggregator/templates/terms.html (hunk 3 above)
Comment (symroe) [reply aggregator/templates/terms.html]: I'm still not convinced / don't know enough, and reckon this e.g is alright

**Comment by chris48s** [comment]
> Are you able to do this?

Done in 2ccb2e0

> I assume he's talking about the text colour?

Looking at the images outside the context of the web page (i.e: they'r transparent images and the backgrounds are applied in CSS, not part of the image), this does make sense. It looks a bit odd on the page, but its not the biggest deal.

**Pull request merged**